docs/user.rst

```diff
@@ -216,6 +216,8 @@ Note:
   make sure you send this device's IPv6 address with the update (myip=...) or
   run the updater on that device and make sure the request originates from
   the IPv6 address you want in DNS.
+* if you want to have all ipv4 addrs to point to the same IP (== router IP), 
+  just use 0 as interface id.
```

Comment:
I thought about this phrasing again: maybe we should not talk about "all" ipv4 addrs here.

You will still need N related hosts entry to get N related host names. So using 0 has nothing to do with making something for "all", but just for that single related hosts entry.

So I suggest to change the phrase to:

if you want to have a related host to point to the same IP v4 as the main host, just use 0 as interface id.

Reply:
If I understand it correctly, this is not a true wildcard domain - you still have to manually specify a list of related hosts - so it does not really fix #133. It can't be used for something like Gitlab Pages which automatically creates subdomains for users and groups - unless I manually add a related host every time somebody creates a new group on Gitlab. Is that correct?

Reply:
yes. you can have many related hosts, but you need 1 entry per hostname.

Reply:
"if you want to have a related host to point to the same IP v4 as the main host, just use 0 as interface id."

sorry, still have some nitpicks:

talking about a router is not necessary, it is just the same ip as the main host.

also, ip addresses to not point to ip addresses. host(name)s point to ip addresses.

Reply:
(to be clear: it is usually the router for typical home networks, but that is not a requirement - it could be also just a single host with a direct internet connection that needs multiple hostnames)